### DISCUSSION TITLE
Is keepassxc using the same configuration file as keepass?

### DISCUSSION THREAD

**Discussion by SvenMarquardt5772** [keepassxreboot/keepassxc#9045] [Q&A] [answered]
I saw, that keepass allows a config file in the form of an XML described here https://keepass.info/help/base/configuration.html. Keepassxc also allows adding a path to a config on the CLI during startup with the --config option.

Are these two the same format with the same features? So, can I use a keepass config with keepassxc?

**Comment by droidmonkey** [comment, marked as answer]
No, you cannot use the keepass config with keepassxc

**Reply by SvenMarquardt5772** [reply]
Is there some sort of documentation what the configuration of keepassxc looks like? I could not find a scheme in the official documentation for that.

**Reply by droidmonkey** [reply]
No because it is not meant to be edited. You can review src/core/config.cpp to discover configuration options and their defaults.
